packages/destination-actions/src/destinations/stackadapt-audiences/index.ts

```diff
@@ -60,10 +60,55 @@ const destination: DestinationDefinition<Settings> = {
       }
     }
   },
+
+  onDelete: async (request, { payload }) => {
+    const userId = payload.userId
+    const formattedExternalIds = `["${userId}"]`
+
+    const syncIds = [sha256hash(String(userId))]
+    const formattedSyncIds = `[${syncIds.map((syncId: string) => `"${syncId}"`).join(', ')}]`
```

Comment:
If I understand correctly, we know that there will always only be one syncId, so why put it into an array and map it if the outcome will always just be `"${syncId}"`

Reply:
Yes that's correct, this was leftover logic from before, I will fix it